pkg/model/notificationevent.proto

```diff
@@ -63,6 +64,12 @@ message NotificationEventDeploymentPlanned {
     repeated string mentioned_groups = 5;
 }
 
+message NotificationEventDeploymentStarted {
+    Deployment deployment = 1 [(validate.rules).message.required = true];
+    repeated string mentioned_accounts = 3;
+    repeated string mentioned_groups = 4;
+}
```

Comment:
I skipped field No. 2 because other codes skipped it.
I don't know why, but I think it's better to do the same as the other codes.